masterlist.yaml

```diff
@@ -396,23 +396,28 @@ plugins:
           - lang: en
             text: '"ccqdrsse001-survivalmode.esl" has a [Known bug](https://www.nexusmods.com/skyrimspecialedition/articles/22/) which is incompatible with "Ordinator - Perks of Skyrim.esp". Consider using the [Unofficial skyrim survival patch](https://www.nexusmods.com/skyrimspecialedition/mods/12655/) to fix this bug.'
   - name: 'Unofficial Skyrim Special Edition Patch.esp'
-    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/266/']
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/266/'
+        name: 'Unofficial Skyrim Special Edition Patch on Nexus Mods'
```

Comment:
These are probably unnecessary, will I keep or revert where name is clear from esp?

Reply:
If the mod name is obvious from the plugin name and there's only one link, I wouldn't both naming the link.

Reply:
Okay I will remove name if there is just one link and the mod name is obvious from the plugin name., I think Arthmoor has mods on Afk & beth as well, I'll probably add those later.